common/oauthTokenManager.go

```diff
@@ -627,7 +394,7 @@ const TokenRefreshSourceTokenStore = "tokenstore"
 
 // OAuthTokenInfo contains info necessary for refresh OAuth credentials.
 type OAuthTokenInfo struct {
-	azcore.TokenCredential
+	azcore.TokenCredential `json:"-"`
 	adal.Token
```

Comment:
What is the purpose of maintaining two tokens?

Reply:
the adal.Token needs to be used until we can entirely get off of Track 1.5 and it's sole purpose is to hold the refresh token for device code cred